bufferserver/src/test/java/com/datatorrent/bufferserver/server/ServerTest.java

```diff
@@ -70,7 +70,7 @@ public static void setupServerAndClients() throws Exception
     eventloopServer.start();
     eventloopClient.start();
 
-    instance = new Server(0, 4096,8);
+    instance = new Server(0, 4096, 8);
```

Comment:
Kind of on a boundary. As this is the only change in the file, I would rather exclude it from a commit. I am not so much concerned with this commit, but I'd prefer to avoid future format changes.